src/toil/utils/toilLaunchCluster.py

```diff
@@ -160,6 +161,16 @@ def main() -> None:
         raise RuntimeError(f'Please provide a value for --zone or set a default in the '
                            f'TOIL_{options.provisioner.upper()}_ZONE environment variable.')
 
+    if options.clusterType is None:
+        logger.warning('Argument --clusterType is not set... using "mesos" as the cluster scheduler. '
+                       'Starting in the next version of Toil, the default cluster scheduler will be '
+                       'set to "kubernetes" if the cluster type is not specified.')
+        options.clusterType = "mesos"
+
+    if options.clusterType == "mesos":
+        logger.warning('You are using a "mesos" cluster, which is no longer recommended as Toil is '
+                       'transitioning to using a kubernetes-based cluster. Consider switching to '
+                       '--clusterType=kubernetes.')
```

Comment:
Should this one go first? Otherwise without the option users get both warnings.

Reply:
I originally intended to show both warnings if the option isn't set, but it might be redundant now that I look at it.